### ISSUE TITLE
Fix examples/counter_isomorphic

### ISSUE THREAD

**Issue by martinfrances107** [gbj/leptos#278]
As it says on the front page 

> NOTE: We're in the middle of merging changes and making fixes to support our upcoming 0.1.0 release. Some of the examples may be in a broken state.

I see examples/counter_isomorphic need some TLC 

I am working on this.

**Comment by gbj** [comment]
It's working fine for me; are you seeing issues or do you mean the warnings?

The warnings are mostly coming from `leptos_router` and are basically there because there are a few features I haven't finished implementing.

**Comment by Bechma** [comment]
When I run the example with `cargo leptos serve` it works, but I found that sometimes panic:
`thread 'actix-rt|system:0|arbiter:3' panicked at 'Tried to access a Runtime that no longer exists.', /home/ubuntu/leptos/leptos_reactive/src/runtime.rs:47:22`

The panic is here:
```rust
pub(crate) fn with_runtime<T>(id: RuntimeId, f: impl FnOnce(&Runtime) -> T) -> T {
    // in the browser, everything should exist under one runtime
    cfg_if! {
        if #[cfg(any(feature = "csr", feature = "hydrate"))] {
            _ = id;
            RUNTIME.with(|runtime| f(runtime))
        } else {
            RUNTIMES.with(|runtimes| {
                let runtimes = runtimes.borrow();
                let runtime = runtimes
                    .get(id)
                    .expect("Tried to access a Runtime that no longer exists."); // <- here
                f(runtime)
            })
        }
    }
}
```

This is produced when you refresh the page in the route `/multi` for SSE counter.

I was wondering if there is another approach that doesn't derive to a panic, maybe to handle the non-existing runtime?

**Comment by gbj** [comment]
Oh thanks I see exactly what's happening. Will fix and will add some docs to clarify for others. Appreciate you pointing it out.

(Basically, `create_signal_from_stream`, unlike `create_resource`, doesn't register with the suspense system so it doesn't cause the HTML stream to stay open. Even though the stream resolves on the first async tick the page has already finished rendering and the runtime used to render it has been disposed. In other words: I used the wrong API for the SSR part of the multi-user component when I wrote the example! Or rather... I closed a memory leak and didn't go back and notice this issue in the example. Either way, appreciated.)
